src/migrations/20240830102144-onboarding-events.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+exports.up = function (db, cb) {
+    db.runSql(
+        `
+            CREATE TABLE onboarding_events_instance (
+                event VARCHAR(255) NOT NULL,
+                time_to_event INTEGER NOT NULL,
+                PRIMARY KEY (event)
+            );
+
+            CREATE TABLE onboarding_events_project (
+               event VARCHAR(255) NOT NULL,
+               time_to_event INTEGER NOT NULL,
+               project VARCHAR(255) REFERENCES projects(id) ON DELETE CASCADE,
```

Comment:
Now we can have foreign key and cascade delete, because we do not need to keep old data.